ansible/templates/topo/t1-64-lag.j2

```diff
@@ -324,35 +324,35 @@
         <ElementType>DeviceInterfaceLink</ElementType>
         <EndDevice>{{ inventory_hostname }}</EndDevice>
         <EndPort>{{ port_alias[32+index*8+2] }}</EndPort>
-        <StartDevice>ARISTA{{ '%02d' % (index+1) }}T0</StartDevice>
+        <StartDevice>ARISTA{{ '%02d' % (index*5+1) }}T0</StartDevice>
```

Comment:
why 5 magic number?

Reply:
![image](https://user-images.githubusercontent.com/2702699/34590821-10993532-f16e-11e7-8819-7b15ee9e5bab.png)

Reply:
in total there are 20 T0 devices connected. we have four groups of links and each of the group has five up links.